src/en/clean-copy/04-Part III. The New Hope/27.md

```diff
@@ -9,48 +9,48 @@ The first half of the 20th century, with all the wars, depressions, and prohibit
+The natural result of the consolidation was that not only the producers' numbers collapsed but also the number of available beer styles as well — e.g. the entire assortment was reduced to pale and dark lagers only (plus occasional unfiltered varieties). Even in Belgium, the market share of lagers was 55% as early as 1928, and since then it increased to 70%[ref:hallen-belgian-industry:23](). In the United States, 101 breweries were producing just two porters in 1970[ref:{"short":"Jankowski, B. (2012)", "extra":"American Porters"}](https://www.morebeer.com/articles/american_porters) and one IPA[ref:steele-ipa:"6":]().
 
-One of the notable exceptions was the United Kingdom, where lagers were losing to bitters and stouts up until the 1990s — although, it didn't prevent the industry from being monopolized: in the 1980s, the six largest brewers produced 75% of all beer in the country and controlled almost every pub[ref](https://publications.parliament.uk/pa/cm200405/cmselect/cmtrdind/128/12805.htm).
+One of the notable exceptions was the United Kingdom, where lagers were losing to bitters and stouts up until the 1990's — although, it didn't prevent the industry from being monopolized. In the 1980's, the UK's six largest brewers produced 75% of all beer in the country and controlled almost every pub[ref](https://publications.parliament.uk/pa/cm200405/cmselect/cmtrdind/128/12805.htm).
 
-One of the moving forces of the lagerization and the consolidation was the incessant process of mergers and acquisitions that had finally culminated in creating the global beer conglomerate AB InBev, which consumed American Annheuser-Busch, Brazilian Ambev (which was in turn the joint venture of Brazilian brewers Antarctica and Brahma), and Belgian Interbrew (which was the result of the Artois-Pidboef merging), joined by SABMiller (which in turn was created after the SAB and Miller deal) in 2016. In some countries, the AB InBev market share exceeds 90% (for example, in Brazil and Uruguay) and even 99% (in the Dominican Republic)[ref](https://www.nbwa.org/resources/look-inside-global-beer-market).
+Unalterable economic forces led to this lagerization and brewery consolidation. Within the new Global markets, the incessant process of mergers and acquisitions had finally culminated in creating the global beer conglomerate AB InBev, which consumed American Annheuser-Busch, Brazilian Ambev (which was in turn the joint venture of Brazilian brewers Antarctica and Brahma), and Belgian Interbrew (which was the result of the Artois-Pidboef merger), joined by SABMiller (which in turn was created after the SAB and Miller deal) in 2016. In some countries, the AB InBev market share exceeds 90% (for example, in Brazil and Uruguay) and even 99% (in the Dominican Republic)[ref](https://www.nbwa.org/resources/look-inside-global-beer-market).
```

Comment:
Economic and societal forces. I guess I feel the consumers will drink what they are told to drink, at least in certain places, China, most of the developing world.

Reply:
I don't want to narrow it down to some specifics, it would be just an oversimplification. There were different forces in different parts of the world. Just the result is the same.